### ISSUE TITLE
Add "Remove" and "Move" functionality

### ISSUE THREAD

**Issue by Xeno-Idaltu** [puddletag/puddletag#804]
### What's missing?

PuddleTag 2.2.0

We are lacking two important and helpful actions.

1) Option to remove files we don't want. Currently, the only option is to delete them.

2) Option to move files to another directory using right click on files we would like to move.

### Your solution

Solution is to bring these two options as they are very helpful.

### Any alternatives?

This functionality is on Mp3Tag

**Comment by sandrotosi** [comment]
> I am rather against including file operations like move, because that's what file managers are for and best at. I would be happy to get rid of the whole delete/trashcan function too and only keep renaming, because that's the only one where you actually need tag data; the rest is pure file management. The tree view even has a right-lick "Open in File Manager" function to make this very easy.

while i agree on this point, sometimes you need the information from puddletag (f.e. bitrate/length etc) to make a decision on "do i keep this file or the other one?", and given the number of feedback we got on the delete/move to trash difference, i feel this is a feature a lot of users are actually using, so removing it may remove something useful.

There's a PR out there for the move-to-trash, im gonna look at it soon.

The other clarification is still required from @Xeno-Idaltu

**Comment by Xeno-Idaltu** [comment]
Yes, having the "unload files" would be very handy for big storage. I tend to open whole directories and as I move along I need to unclog my view by "unloading" files by tag.

I think the "delete" option should stay but as an option for the "right click menu" as I have accidentally deleted files when trying to open it's properties.

I currently use KDE's Dolphin Manager and its hard to move files by tags. Dolphin's columns currently only shows 7 possible options for tags: Artists, Album, Genre, Duration, etc. The side effect is that it is very slow and tagging moving only for one directory at a time. We can move multiple files from different directories but by file name. 

In my case I want to move all files tagged type="intrumental" into one major directory and from there create subdirectories based on mood like "uplifting", "melancholical", etc. If were to open my whole database using PuddleTag, this is where the function of "unload" comes handy as files containing tag type="instrumental" are all over the place. Once I filter the system, I would just unload files without having to delete them. I would then highlight files containing tag mood="uplifitng", right click on them and move all those files to /mymusicfiles/instrumental/uplifting

**Comment by audiomuze** [comment]
You can already move files as long as it’s on the same physical device - just put the file path and folder name you want in the toolbar, select all the files you want to move and hit R

**Comment by Visual-Dawg** [comment]
I dont understand how moving works. 
What toolbar do you mean? 
A context-menu option for that would be so nice

**Comment by vdawg-git** [comment]
Bump because I still dont get how moving works :/

**Comment by audiomuze** [comment]
include the path to where you want to move the files to in the settings in the toolbar:
![image](https://github.com/user-attachments/assets/6fb54d17-c714-435f-9c55-c2c724dc840f)

e.g.:

![image](https://github.com/user-attachments/assets/bd65de66-314e-4f2c-a91b-4e4890a52614)


then hit the rename icon on the right side of the same toolbar.

That'll cause the selected folders to be renamed and moved to /tmp/.

Remember, the target directory MUST be on the same disc as the source.

**Comment by vdawg-git** [comment]
Bumping as this would greatly simplify my workflow.
I have two directories, one for my tagged music and one for my not-fully tagged music.
It is so much nicer to just move files which are done away.

Edit: Renaming the files to move them feels a bit like a hack and it is somewhat hard to see if it worked properly. And you need to remember the full file-paths.

**Comment by audiomuze** [comment]
> Edit: Renaming the files to move them feels a bit like a hack and it is somewhat hard to see if it worked properly. And you need to remember the full file-paths.

that is exactly what moving files is ... renaming them (unless you're trying to move them to a different physical drive).  if you want to see that it's worked just include __dirpath in your column view and you'll see their location immediately and visually confirm they did indeed move.  The other advantage is you can then sort by filename, then __dirpath, meaning all files located in the same path appear grouped together, making it easy to focus on the stuff you're still tagging.

<img width="704" height="164" alt="Image" src="https://github.com/user-attachments/assets/96e8d299-77ef-4c0f-9c86-ba379b33b203" />
